### ISSUE TITLE
Implement due only filter toggle on main register page

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-fp#8]
- [x] On the main register page, there is a "Due only" toggle button that filters the list of clients to only those that have a visit scheduled (either due, overdue, or upcoming).
- [x] The default state is not filtered. So all records are listed in the default-sort pattern only. 
- [x] When the toggle is activated, the list filters to only those records with a visit scheduled. We still use the default sort for the filtered list.
- [x] The filter toggle is kept active until the user toggles it back to see the full list, or until they exit the app.
- [x] Active toggle state is shown below. Please grab the yellow color code from the ANC app.

<img width="126" alt="Screen Shot 2020-03-25 at 6 18 25 PM" src="https://user-images.githubusercontent.com/3488986/77571326-12b9ca80-6ec5-11ea-96b3-044319418a8f.png">

**Comment by dubdabasoduba** [comment]
@cafootitt I don't think we have the yellow color code from ANC I will try check if the CHW project has it.

**Comment by cafootitt** [comment]
@dubdabasoduba You're right, I meant to put wcaro app. Did you get the yellow code already? If not, I can ask Ronald for it.

**Comment by cafootitt** [comment]
QA results 9 June:

- [x] Due only filter doesn't seem to be working. See below. There are several clients registered with an upcoming visit scheduled (Gimmy Force, Client Name Client Last, for example), yet when the due only filter is on, the client list is empty.

![Screenshot_20200609-094045_Family Planning](https://user-images.githubusercontent.com/3488986/84126468-d6aef400-aa35-11ea-875c-b0723567daad.jpg)
![Screenshot_20200609-094106_Family Planning](https://user-images.githubusercontent.com/3488986/84126472-d7e02100-aa35-11ea-8408-7eecc283383f.jpg)
